docs/source/contributing/developing.adoc

```diff
@@ -111,18 +111,18 @@ $ glide install -v
 ----
 
 [[update-dependencies]]
-=== Updating dependencies
+=== Updating Dependencies
 
 If your work requires a change to the dependencies, you need to update the Glide configuration.
 
 . Edit *_glide.yaml_* to change the dependencies as needed.
 
-. Delete *_glide.lock_*  and re-create the vendor directory by running `make vendor`.
+. Delete *_glide.lock_* and re-create the vendor directory by running `make vendor`.
  Glide will recognize that there is no lock file and recalculate the required dependencies.
 
-. Check-in the updated *_glide.yaml_* and *_glide.lock_*.
+. Check-in the updated *_glide.yaml_* and *_glide.lock_* files.
```

Comment:
> And a general comment, even though you personally use certain language to discuss things, try to remember that not everyone does, and we need to be prepared for different types of readers

Agree. On the same note, please remember that we are targeting developers who have partly their own lingo. What might make sense for a non technical user or non-dev, will sound awkward for a developer.